### PR TITLE
Updated paper 15 for Avatars by adding the code repo link of v2 and their new title.

### PR DESCRIPTION
**Pull request by ElenaViewSynthesis**
See descriptions of my 2 previous commit changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -705,7 +705,12 @@ We present GaussianAvatar, an efficient approach to creating realistic human ava
...
 </details>
 
- [📄 Paper](https://arxiv.org/pdf/2312.02134.pdf) | [🌐 Project Page](https://ustc3dv.github.io/FlashAvatar/) | [💻 Code (not yet)]()
+ [📄 Paper v1](https://arxiv.org/pdf/2312.02214v1) | [🌐 Project Page](https://ustc3dv.github.io/FlashAvatar/) | [💻 Code (not yet)]()
+ 
+  ### FlashAvatar: High-fidelity Head Avatar with Efficient Gaussian Embedding (v2)
+
```
Comment (MrNeRF) [inline README.md:711]: Thank you for your contribution. I am just wonder why we should present both versions? I would suggest the latest is the greatest and would remove the old one? The rest looks great!

**Review by MrNeRF — CHANGES_REQUESTED** [review]
Thx for your contribution. Please see my comment

**Review by ElenaViewSynthesis — COMMENTED** [review]
Yes, we should only keep the latest version.

**Comment by MrNeRF** [comment]
Thank you :)

**Pull request merged**